scalalib/overrides-2.11/scala/collection/immutable/Range.scala

```diff
@@ -513,7 +524,10 @@ object Range {
   // indefinitely, for performance and because the compiler seems to bootstrap
   // off it and won't do so with our parameterized version without modifications.
   object Int {
-    def apply(start: Int, end: Int, step: Int) = NumericRange(start, end, step)
-    def inclusive(start: Int, end: Int, step: Int) = NumericRange.inclusive(start, end, step)
+    def apply(start: Int, end: Int, step: Int): NumericRange.Exclusive[Int] =
+      NumericRange(start, end, step)
+
+    def inclusive(start: Int, end: Int, step: Int): NumericRange.Inclusive[Int] =
```

Comment:
Should this return just `NumericRange[Int]`?

Reply:
Scaladoc has `NumericRange.Inclusive` signature: https://www.scala-lang.org/api/2.12.11/scala/collection/immutable/NumericRange$.html